ndt7.go

```diff
@@ -53,6 +53,14 @@ type testFn = func(
 // by NewClient in the Client.Dialer.HandshakeTimeout field.
 const DefaultWebSocketHandshakeTimeout = 7 * time.Second
 
+// TestData contains the latest Measurement send by the server and the client,
+// plus the latest ConnectionInfo sent by the server.
+type TestData struct {
```

Comment:
I don't quite like this name because I find it too generic. How about LatestMeasurement? (My understanding is that this data structure is that and, after we fix server, we wont need separate ConnectionInfo)

Reply:
Your understanding is correct: if ndt-server started sending `ConnectionInfo` with every measurement, rather than just on the first one, then we could theoretically remove the ConnectionInfo field here and just keep the latest Measurement.

However, I feel like we should not rely on the ndt-server specific implementation but rather follow the protocol specification, which at the moment goes in the opposite direction:
```
Servers MUST send this message exactly once.
Clients SHOULD cache the first received instance of this message, and discard any subsequently received instance of this message.
```
To clarify: I like the idea of sending ConnectionInfo every time, I just think we should also update the specification.

Reply:
Yeah, what you say is sensible.